.github/workflows/hqs-build-deploy-rust-pyo3-prerelease.yml

```diff
@@ -1,11 +1,31 @@
-name: HQS build and deploy for rust pyo3 repos
+name: HQS build and deploy for rust pyo3 repos (prerelease)
 
 on:
   workflow_dispatch:
   release:
-    types: [created]
+    types: [prereleased]
 
 jobs:
+  api_integration:
+    runs-on: ubuntu-latest
+    env:
+      QRYD_API_HQS: 1
+    continue-on-error: true
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          default: true
+          components: rustfmt
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+      - run: |
+          python -m pip install numpy
+          cargo test --workspace --no-default-features --locked
+
```

Comment:
Is that set so that it may fail already?

Reply:
`continue-on-error`, but actually I need to move it to the internal steps, otherwise the workflow won't be green when it fails.